README.md

```diff
@@ -206,6 +206,20 @@ stored in [Cloud Storage][cloud-storage].
 
 ## Integrations
 
+- **App Engine (Flex)** - When invoked via [App Engine Flex][app-engine-flex],
+  Berglas resolves environment variables to their plaintext values using the
+  [`berglas://reference syntax][reference-syntax]. This integration works with
+  any language runtime because berglas serves as the entrypoint to the Docker
+  container. See [examples/appengineflex](examples/appengineflex) for examples
+  and invocations.
+
+- **App Engine** - When invoked via [App Engine][app-engine], Berglas resolves
```

Comment:
Can you call this "App Engine Standard" instead so it's clearer? You can keep the URLs and stuff as-is, I just want to optimize for greppability.